sites/feedback/ajax/page-feedback-fr.html

```diff
@@ -0,0 +1,74 @@
+<div class="gc-pft row" data-wb-jsonmanager='{
+	"name": "feedback",
+	"extractor": [
+		{ "interface": "locationHref", "path": "submissionPage" },
+		{ "selector": "title", "path": "pageTitle" },
+		{ "selector": "html", "attr": "lang", "path": "language" },
+		{ "selector": "meta[name=\"dcterms.type\"]", "attr": "content", "path": "themeopt" },
+		{ "selector": "meta[name=\"dcterms.subject\"]", "attr": "content", "path": "sectionopt" },
+		{ "selector": "meta[name=\"author\"]", "attr": "content", "path": "institutionopt" }
+	]
+}'>
+	<div class="col-sm-10 col-md-9 col-lg-8">
+		<section class="well mrgn-bttm-0" aria-live="polite">
+			<h3 class="wb-inv">Rétroaction sur la page</h3>
+			<form id="gc-pft-frm" action="https://pagesuccessemailqueue.azurewebsites.net/api/QueueProblemForm" method="post" class="wb-postback" data-wb-postback='{"success":".gc-pft-thnk", "content":"#gc-pft-frm"}'>
```

Comment:
```suggestion
			<form action="https://pagesuccessemailqueue.azurewebsites.net/api/QueueProblemForm" method="post" class="wb-postback" data-wb-postback='{"success":".gc-pft-thnk"}'>
```

Ditto for the French version.